scripts/ci/images/fedora-msan/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG INSTALL_PREFIX=/opt/msan
+ARG TOOLCHAIN_FILE=toolchain.cmake
+ARG CFLAGS="-fsanitize=memory -fsanitize-memory-track-origins"
```

Comment:
Do we need CXXFLAGS set here too?

Reply:
Might not hurt to add it, but I haven't had an explicit need for it yet. In this context, `CFLAGS` and `CXXFLAGS` are only needed for projects that can't be built with CMake.